src/Test/MakerTestEnvironment.php

```diff
@@ -405,7 +398,7 @@ private function determineMissingDependencies(): array
         ');
 
         $process = MakerTestProcess::create('php dep_runner.php', $this->path)->run();
-        $data = json_decode($process->getOutput(), true);
+        $data = json_decode($process->getOutput(), true, 512, \JSON_THROW_ON_ERROR);
         if (null === $data) {
             throw new \Exception('Could not determine dependencies');
         }
```

Comment:
I think we don't need this exception anymore, right?

Reply:
good catch